README.md

```diff
@@ -73,6 +72,18 @@ Name                   | Description
 `requireBlueprintName` | Require parsed blueprints have a title (default: false)
 `generateSourceMap`    | Enable sourcemap generation (default: false)
 
+
+## Improving performance
+Optionally you can also install C++ bindings to [Drafter](https://github.com/apiaryio/drafter) - [Protagonist](https://github.com/apiaryio/protagonist).
+
+```bash
+npm i protagonist
```

Comment:
I'd prefer if we used `install` instead of `i` especially in public facing documentation. It's clearer to users who are not too familiar with these commands.